spec/git_merge_po_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'tempfile'
+require 'pry'
+
+def example_title
+  RSpec.current_example.description
+end
+
+# a trivial PO file loader. returns a hash.
+def load_po(data)
+  h = {}
+  state = :none # :id, :str
+  id = ""
+  str = ""
+  (data.split("\n") + ['']).each do |line|
+
+    case state
+    when :none
+      next if line =~ /^$|^#/
+      fail unless line =~ /^msgid "(.*)"$/
+      id = $1
+      state = :id
+    when :id
+      if line =~ /^msgstr "(.*)"/
+        state = :str
+        str = $1
+        next
+      end
+      fail unless line =~ /^"(.*)"$/
+      id += $1
+    when :str
+      if line =~ /^$|^#/
+        state = :none
+        h[id] = str
+        next
+      end
+      fail unless line =~ /^"(.*)"$/
+      str += $1
+    end
+  end
+  h
+end
+
+describe 'git merge-po' do
+  let(:subject) { @subject }
```

Comment:
this can be written as `subject { @subject }` but tbh is is that slow if you execute the before(:all) on a before each? Just asking because if the before block as a method instead you could say `subject { loaded_po }` or something.